tests/unit/utils/test_component_tools.py

```diff
@@ -129,10 +55,12 @@ def test_search_by_filepath(self):
                 'lib/ansible/modules/network/ios/ios_facts.py',
                 'lib/ansible/modules/network/ios/ios_facts.py'
             ],
-            'json_query': [
-                'lib/ansible/plugins/filter/json_query.py',
-                'lib/ansible/plugins/filter/json_query.py'
-            ],
+            # Doesn't work, lib/ansible/modules/network/nso/nso_query.py is
```

Comment:
So has this never worked?

Reply:
It works with [2017-10-24 Ansible source tree](https://github.com/ansible/ansibullbot/blob/fae5f7c4af5d170ef101fe58a284a61ea7bca8e8/tests/unit/utils/test_component_tools.py#L57): `nso_query` module was added after ([2017-12-22](https://github.com/ansible/ansible/pull/34188)).

I would rather use an up to date tree (meaning `devel` branch) rather than using an out of date tree. Advantage: new failures like this one are detected.

The next step could be to:
* mock either the `ansible-doc` command or the ansible repository content
* add integration test in order to check that ansibullbot is able to call `ansible-doc` (in order to avoid problem like e52210adea8032d996ff333322b6a4922e01e6ee)

Reply:
@pilou- can you please create an issue for the next step?